samples/msgext-northwind-inventory-csharp/TeamsApp/teamsapp.local.yml

```diff
@@ -0,0 +1,86 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.2/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.2
+
+additionalMetadata:
+  sampleTag: Microsoft-Teams-Samples:msgext-northwind-inventory-csharp-csharp
+
+provision:
+  - uses: aadApp/create # Creates a new Azure Active Directory (AAD) app to authenticate users if the environment variable that stores clientId is empty
+    with:
+      name: msgext-northwind-inventory-csharp-aad # Note: when you run aadApp/update, the AAD app name will be updated based on the definition in manifest. If you don't want to change the name, make sure the name in AAD manifest is the same with the name defined here.
+      generateClientSecret: true # If the value is false, the action will not generate client secret for you
+      signInAudience: "AzureADMultipleOrgs" # Multitenant
+    writeToEnvironmentFile: # Write the information of created resources into environment file for the specified environment variable(s).
+      clientId: AAD_APP_CLIENT_ID
+      clientSecret: SECRET_AAD_APP_CLIENT_SECRET # Environment variable that starts with `SECRET_` will be stored to the .env.{envName}.user environment file
+      objectId: AAD_APP_OBJECT_ID
+      tenantId: AAD_APP_TENANT_ID
+      authority: AAD_APP_OAUTH_AUTHORITY
+      authorityHost: AAD_APP_OAUTH_AUTHORITY_HOST
+
+  # Creates a Teams app
+  - uses: teamsApp/create
+    with:
+      # Teams app name
+      name: msgext-northwind-inventory-csharp-${{TEAMSFX_ENV}}
+    # Write the information of created resources into environment file for
+    # the specified environment variable(s).
+    writeToEnvironmentFile: 
+      teamsAppId: TEAMS_APP_ID
+
+  - uses: script
+    with:
+      run:
+        # echo "::set-teamsfx-env MICROSOFT_APP_TYPE=SingleTenant";
+        echo "::set-teamsfx-env MICROSOFT_APP_TYPE=MultiTenant";
+        echo "::set-teamsfx-env MICROSOFT_APP_TENANT_ID=${{AAD_APP_TENANT_ID}}";
```

Comment:
This step seems unnecessary, what is it used for?

Reply:
This part is a TTK auto generated code, It can be edited and directly we can add value "AAD_APP_TENANT_ID" and "MultiTenant" to the appsettings part in next line.

Due to naming conventions looked fine we left as it is. Please let me know if you want me to edit it.